documentation/operations/archival-process.md

```diff
@@ -0,0 +1,16 @@
+# Archival Process
+
+To archive the instances you will need to connect to the Ocim server using ssh and use the Django shell to archive instances since there is currently no UI to do so. Please, don't wait for the archival process for more than 5 minutes once it's running; this takes some hours to run, so it's better to check back later.
+
+*  Connect to Ocim using: `ssh ubuntu@ocim.net.opencraft.hosting`.
```

Comment:
This is public documentation so I think it's better to use an example domain.

Reply:
Done